action-scripts/brownie/scripts/validate_bip.py

```diff
@@ -42,16 +43,37 @@ def validate_chain_specified(file: dict) -> Tuple[bool, str]:
     Validates that chain is specified in file
     """
     chain = file.get('chainId')
-    if not chain or not isinstance(chain, str):
-        return False, "No chain specified or it is not a string"
+    chains = list(AddrBook.CHAIN_IDS_BY_NAME.values())
+    if int(chain) not in chains:
+        return False, f"No chain specified or is not found in known chain list: {chain} in {chains}"
     return True, ""
 
+def validate_txs_have_extra_data(file: dict) -> Tuple[bool, str]:
```

Comment:
I can't understand why do we need to validate BIP payload that you've added yourself...